server/src/main/java/org/spine3/server/entity/FieldMasks.java

```diff
@@ -197,7 +197,8 @@ public static FieldMask maskOf(Descriptor typeDescriptor, int... fieldTags) {
                 builder.setField(field, wholeMessage.getField(field));
             }
         }
-        @SuppressWarnings("unchecked")       // It's fine as the constructor is of {@code MessageCls.Builder} type.
+        @SuppressWarnings("unchecked") // It's fine as the constructor
```

Comment:
Let's write a comments, which are too long, below a suppression. Check the similar cases.